networkdb/cluster.go

```diff
@@ -107,7 +107,6 @@ func (nDB *NetworkDB) clusterInit() error {
 		config.BindPort = nDB.config.BindPort
 	}
 
-	config.ProtocolVersion = memberlist.ProtocolVersionMax
```

Comment:
In a future version of memberlist `DefaultLANConfig` can set a version higher than 3. Even if there is a Release Note stating this change unless we read that and set the version explicitly it will break us when we go to that version. Instead if we set it to `ProtocolVersion2Compatible` now it makes our code future proof independent of what memberlist version change is, right ? Do you see a downside in setting the version explicitly ?